### PR TITLE
Android webview doc. RD-20965

### PR DESCRIPTION
**Pull request by brenauj**

### CODE REVIEW

File: docs/interactions/structured-messages/webview-sdk.md [hunk 1]
```diff
@@ -89,13 +89,13 @@ Since not every channel supports every feature, you'll find support info in the
...
 
 ## Integrations specific behavior
 
 Some integrations have extra behavior specific to their related channel. Those are the following.
 
-### Engage Messaging Web/iOS
+### Engage Messaging
 
-When opening your webview page in Engage Messaging web chat or in Engage Messaging iOS, the title element's text will become the chat webview header's text, and when you change your page's title element's `textContent`, it will change the chat's webview header text.
+When opening your webview page in Engage Messaging, the title element's text will become the chat webview header's text, and when you change your page's title element's `textContent`, it will change the chat's webview header text.
```
Comment (brenauj) [inline docs/interactions/structured-messages/webview-sdk.md:101]: Screenshot from 2022-03-18 16-05-25

File: docs/interactions/structured-messages/webview.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ The page on the webview should be hosted on your side, and setup to accept displ
 | | **webview** | **webview_height** |
 |-|-|-|
 | **Apple Business Chat** | N/A | N/A |
-| **Engage Android Messaging** | **<span style="color:orange">Not yet</span>** | **<span style="color:orange">Not yet</span>** |
+| **Engage Android Messaging** | **<span style="color:green">Yes, since v2.3.1</span>** | **<span style="color:green">Yes, since v2.3.1</span>** |
```
Comment (brenauj) [inline docs/interactions/structured-messages/webview.md:32]: Screenshot from 2022-03-18 16-04-20

File: docs/interactions/structured-messages/webview.md [hunk 3]
```diff
@@ -43,16 +43,24 @@ The page on the webview should be hosted on your side, and setup to accept displ
...
 |-|-|-|-|
 | **Preview** | <img class="img-fluid" width="419" src="../../../img/ios-messaging-webview-full.jpg"> | <img class="img-fluid" width="419" src="../../../img/ios-messaging-webview-tall.jpg"> | <img class="img-fluid" width="419" src="../../../img/ios-messaging-webview-compact.jpg"> |
+
+## Engage Messaging Android customization
+
+Engage Messaging Android allows you to set a smaller window to open. To do so, set the `webview_height` parameter to fit your need, you can see the design for `full`, `tall` and `compact` webviews in the following visuals.
+
+| **Value** | **`full` (default)** | **`tall`** | **`compact`** |
+|-|-|-|-|
+| **Preview** | <img class="img-fluid" width="419" src="../../../img/android-messaging-webview-full.jpg"> | <img class="img-fluid" width="419" src="../../../img/android-messaging-webview-tall.jpg"> | <img class="img-fluid" width="419" src="../../../img/android-messaging-webview-compact.jpg"> |
```
Comment (brenauj) [inline docs/interactions/structured-messages/webview.md:66]: Screenshot from 2022-03-18 15-57-36

**Review by DrissTM — APPROVED** [review]
LGTM :+1: (and thanks for the screenshots 🙏)

**Pull request merged**